src/components/shared/transactionResult/statusConfig.js

```diff
@@ -0,0 +1,92 @@
+import { transactionToJSON } from '@utils/transaction';
+import { isEmpty } from '@utils/helpers';
+
+export const txStatusTypes = {
+  multisigSignaturePartialSuccess: 'MULTISIG_SIGNATURE_PARTIAL_SUCCESS',
+  multisigSignatureSuccess: 'MULTISIG_SIGNATURE_SUCCESS',
+  signatureSuccess: 'SIGNATURE_SUCCESS',
+  multisigBroadcastSuccess: 'MULTISIG_BROADCAST_SUCCESS',
+  broadcastSuccess: 'BROADCAST_SUCCESS',
+  signatureError: 'SIGNATURE_ERROR',
+  broadcastError: 'BROADCAST_ERROR',
+  hwRejected: 'HW_REJECTED',
+};
+
+export const statusMessages = t => ({
+  [txStatusTypes.multisigSignaturePartialSuccess]: {
+    title: t('Your signature was successful'),
+    message: t('You can download or copy the transaction and share it with other members.'),
+  },
+  [txStatusTypes.multisigSignatureSuccess]: {
+    title: t('Your signature was successful'),
+    message: t('You can download or copy the transaction and share it with other members. You can send the transaction too.'),
+  },
+  [txStatusTypes.signatureSuccess]: {
+    title: t('Submitting the transaction'),
+    message: t('Your transaction is being submitted to the blockchain.'),
+  },
+  [txStatusTypes.multisigBroadcastSuccess]: {
+    title: t('Transaction submitted'),
+    message: t('Your transaction has been submitted and will appear in sender account\'s wallet after confirmation.'),
+  },
+  [txStatusTypes.broadcastSuccess]: {
+    title: t('Transaction submitted'),
+    message: t('Your transaction has been submitted and will be confirmed in a few moments.'),
+  },
+  [txStatusTypes.signatureError]: {
+    title: t('Transaction failed'),
+    message: t('Oops, it looks like something went wrong. Please try again.'),
+  },
+  [txStatusTypes.broadcastError]: {
+    title: t('Transaction failed'),
+    message: t('Oops, it looks like something went wrong. Please try again.'),
+  },
```

Comment:
the title and message should be different ? instead of `Oops, it looks like something went wrong. Please try again` we can say `Error occurred during transaction signing process. Please try again.`?
same for `Error occurred during transaction broadcasting process. Please try again.`?